### ISSUE TITLE
USER ISSUE: Exception while chopping stump

### ISSUE THREAD

**Issue by Fazamere** [StrangeLoopGames/EcoIssues#6393]
**Version:** 0.7.2.5 beta

**Steps to Reproduce:**
Wearing Steel Axe, chopped a tree, and crashed.
Our server had a major issue this morning where all the backups for yesterday were corrupted. We had to rollback to 2 days ago.	Related?	

**Expected behavior:**
Chopped the stump.

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60.00%>Object reference not set to an instance of an object.


Stack:
   at Eco.Simulation.Agents.AI.AITarget.<UpdatePosition>b__32_0()
   at Eco.Shared.Utils.LockUtils.TryLock(Object lockObject, Action ifLockAcquired, Int32 millisecondsTimeout)
   at Eco.Simulation.Agents.AI.AITarget.UpdatePosition()
   at Eco.Simulation.Agents.AI.AITarget.get_Position()
   at Eco.Shared.Networking.NetObjectExtensions.Priority(INetObject netObject, INetObjectViewer viewer)
   at System.Linq.EnumerableSorter`2.ComputeKeys(TElement[] elements, Int32 count)
   at System.Linq.EnumerableSorter`1.Sort(TElement[] elements, Int32 count)
   at System.Linq.OrderedEnumerable`1.<GetEnumerator>d__1.MoveNext()
   at System.Linq.Enumerable.<TakeIterator>d__25`1.MoveNext()
   at Eco.Shared.Networking.NetObjectManager.GetInitialObjectStates(IEnumerable`1 newObjects, INetObjectViewer viewer, ICollection`1 monitored)
   at Eco.Plugins.Networking.Clients.Client.Update(Stopwatch timer)
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_1()
</size>
```

**Comment by MarjieAVolk** [comment]
Similar to https://github.com/StrangeLoopGames/EcoIssues/issues/5871

**Comment by SeanAtr0n** [comment]
fixed 7.4
